### PR TITLE
android/ui: show count of Mullvad countries, also fix navigation-rela…

### PR DESCRIPTION
**Pull request by oxtoacart**
…ted crash when adding new account

Updates tailscale/corp#18202

### CODE REVIEW

File: android/src/main/java/com/tailscale/ipn/MainActivity.kt [hunk 1]
```diff
@@ -106,7 +105,7 @@ class MainActivity : ComponentActivity() {
 
     setContent {
       AppTheme {
-        val navController = rememberNavController()
+        navController = rememberNavController()
```
Comment (oxtoacart) [inline android/src/main/java/com/tailscale/ipn/MainActivity.kt:108]: This is a drive-by fix to avoid errors like 

```
kotlin.UninitializedPropertyAccessException: lateinit property navController has not been initialized
                                                                                                    	at com.tailscale.ipn.MainActivity.onNewIntent(MainActivity.kt:248)                                                                                               	
```

File: android/src/main/java/com/tailscale/ipn/MainActivity.kt [hunk 2]
```diff
@@ -163,18 +162,16 @@ class MainActivity : ComponentActivity() {
...
-                    composable(
-                        "mullvad/{countryCode}",
-                        arguments =
-                            listOf(navArgument("countryCode") { type = NavType.StringType })) {
-                          MullvadExitNodePicker(
-                              it.arguments!!.getString("countryCode")!!, exitNodePickerNav)
-                        }
-                    composable("runExitNode") { RunExitNodeView(exitNodePickerNav) }
-                  }
+                  composable("exitNodes") { ExitNodePicker(exitNodePickerNav) }
```
Comment (oxtoacart) [inline android/src/main/java/com/tailscale/ipn/MainActivity.kt:165]: This is a drive-by fix to broken back-navigation from Mullvad to exit nodes

**Review by kari-ts — APPROVED** [review]
